pom.xml

```diff
@@ -55,6 +55,21 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
Did you update Marge with the versions that have been added? Checked against whitelists, etc

Reply:
Yeah and if we have to submit the swaps, we might as well use the latest version available (looks like there's a 1.2.5 release that might be good to move to if it still works with our other dependencies)